cpp/src/hash/hashing.cu

```diff
@@ -765,6 +771,23 @@ std::unique_ptr<column> murmur_hash3_32(table_view const& input,
   return output;
 }
 
+std::unique_ptr<column> sha1(table_view const& input,
+                             rmm::mr::device_memory_resource* mr,
+                             cudaStream_t stream) {
+}
+
+std::unique_ptr<column> sha256_base(table_view const& input,
+                                    bool truncate_output,
```

Comment:
Don't use a bool parameter. Use an enum class with a descriptive name so it's more obvious at the callsite what this controls.